src/Files.App/Views/Layouts/ColumnsLayoutPage.xaml.cs

```diff
@@ -96,7 +100,7 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
 
 			if (!string.IsNullOrEmpty(pathRoot))
 			{
-				var rootPathList = App.QuickAccessManager.Model.PinnedFolders.Select(NormalizePath)
+				var rootPathList = WindowsQuickAccessService.Folders.Select(x => NormalizePath(x.Path))
```

Comment:
This should only select from the pinned folders.

Reply:
I resolved this in [ba61c81](https://github.com/files-community/Files/pull/16485/commits/ba61c817a810feac980c925a38e39ce47d3ee48b)